packages/@coorpacademy-components/src/molecule/cm-popin/index.js

```diff
@@ -82,12 +82,12 @@ const CMPopin = props => {
             </div>
```

Comment:
Not related to the changes made in this pr, but it would be nice to fix a couple of lines:78 & 58 have their datanames on interpolation strings formats `data-name={`cm-popin-cta`}` instead of just using single quotes `data-name={'cm-popin-cta'}`